src/create_features.py

```diff
@@ -0,0 +1,167 @@
+# input: a pandas dataframe
+# output: pickle
+
+import time
+start = time.time()
+#from classifiers import *
+from get_data import *
+from nltk.stem import WordNetLemmatizer
+wordnet_lemmatizer = WordNetLemmatizer()
+#from stanford_polite import *
+from convo_politeness import *
+from functools import partial
+import TextParser
+import text_cleaning
+import text_modifier
+from util import *
+import argparse
+import config
+import json
+import logging
+import multiprocessing as mp
+import pandas as pd
+import pickle
+import requests
+import sys
+nlp = spacy.load("en_core_web_md",disable=["parser","ner"])
```

Comment:
Personally, I find the lack of whitespace less readable. I don't know what style guides say

Reply:
Same.. this is old code.. fixed..